Makefile

```diff
@@ -45,13 +52,26 @@ venvs/ot2: ot2monorepoClone
 	popd && \
 	deactivate
 
-# OVERRIDE_SETTINGS_DIR must be set to use opentrons v3
-# (otherwise it will try to access /data dir during 'import opentrons')
+.PHONY: parse-ot1
+parse-ot1: venvs/ot1 $(OT1_OUTPUT_FILES)
+
+# Parse all OT1 python files
+$(BUILD_DIR)/%.ot1.py.json: protocols/%.ot1.py
+	mkdir -p $(dir $@) && \
+	source venvs/ot1/bin/activate && \
+  python protolib2/parse/parseOT1.py $< $@ && \
+	deactivate
+
 .PHONY: parse-ot2
-parse-ot2: venvs/ot2
+parse-ot2: venvs/ot2 $(OT2_OUTPUT_FILES)
+
+# Parse all OT2 python files
+# Note: OVERRIDE_SETTINGS_DIR must be set to use opentrons v3
+$(BUILD_DIR)/%.ot2.py.json: protocols/%.ot2.py
+	mkdir -p $(dir $@) && \
```

Comment:
Consider adding an `@` in front of this `mkdir` to avoid echoing it to the console (to make things a little less chatty)